### PR TITLE
Added Random Toolkit Tournament manual preset.

### PR DESCRIPTION
**Pull request by czajowaty**

### CODE REVIEW

File: curry_bot.py [hunk 1]
```diff
@@ -97,70 +88,7 @@ async def bingo(ctx):
...
-                preset = key
-                matches += 1
-        if matches != 1:
-            preset = NULL_PRESET
-    return preset
-
-
-def make_seed(preset, seed):
-    return "{}?P:{},,{}".format(RANDO_BASE, preset, seed)
+    await RandoCommandHandler(ctx, args).handle()
```
Comment (HexTree) [inline curry_bot.py]: I'm attempting to keep to the pattern that 'curry_bot.py' handles *all* Discord messaging stuff, and the imported modules are not allowed to (perhaps I've violated that rule in a couple of places, but will try to correct that). The advantage of this would be separation of concerns: that we keep all the Discord IO in one place (this file, or a dedicated Discord wrapper module in the future), so that CurryBot isn't saying anything we don't expect. Also, it helps to keep the modules bot-agostic, i.e. they don't care about the nature of the bot, and don't use things like Curry emotes.

TLDR curry_bot cares only about Discord, and ad_rando cares only about adrando interpretation.

To follow this pattern, maybe it is best for RandoCommandHandler to return an iterable of strings, and then here in curry_bot we take that list and create the messages.

File: ad_rando/seed_generator.py [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+                in AdRandomizerParamsDescriptorSelector.all_presets().items()))
+
+    async def _generate_and_print_seeds(self, preset_name):
+        try:
+            params_descriptor = AdRandomizerParamsDescriptorSelector.select(preset_name)
+            description, params = params_descriptor
+            seeds_number = self._parse_seeds_number()
+            await ctx.send(curry_message(f"Generating {seeds_number} {description} seeds..."))
+            RandoCommandHandler.RANDO_LINKS = SeedGenerator(params, seeds_number).generate()
+            await ctx.send(curry_message("Rando seed links updated"))
```
Comment (HexTree) [inline ad_rando/seed_generator.py]: As discussed, try to cut down to only one 'ctx.send', or otherwise as small a number as possible, being performed in curry_bot.py.

File: curry_bot.py (hunk 1 above)
Comment (czajowaty) [reply curry_bot.py]: Fixed in the latest commit.

File: ad_rando/seed_generator.py (hunk 2 above)
Comment (czajowaty) [reply ad_rando/seed_generator.py]: Fixed in the latest commit.

**Review by HexTree — APPROVED** [review]
